### PR TITLE
Create the board

### PR DESCRIPTION
**Pull request by shvshv44**

### CODE REVIEW

**Review by Guyeise1 — APPROVED** [review]

File: client/src/app/board-cell/board-cell.component.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import {Component, Input, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-board-cell',
+  templateUrl: './board-cell.component.html',
+  styleUrls: ['./board-cell.component.scss']
+})
+export class BoardCellComponent implements OnInit {
+
+  @Input()
```
Comment (SharonEizner) [inline client/src/app/board-cell/board-cell.component.ts:10]: change it to be in one line 
@Input() row

File: client/src/app/board-cell/board-cell.component.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  col = -1;
+
+  @Input()
+  playerCode = -1; // if a player stands on the cell
+
+  constructor() { }
+
+  ngOnInit(): void {
```
Comment (SharonEizner) [inline client/src/app/board-cell/board-cell.component.ts:21]: delete

File: client/src/app/board-wall/board-wall.component.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<div class="wall" [ngClass]="(isHorizontal)?'horizontal':'vertical'">
```
Comment (SharonEizner) [inline client/src/app/board-wall/board-wall.component.html:1]: wall need to be 2*1 and not 1*1

File: client/src/app/app.component.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#game-screen {
+  height: 100%;
+  width: 100%;
+  background-color: #f1ce7b;
```
Comment (SharonEizner) [inline client/src/app/app.component.scss:4]: move it to styles.scss.
create a var and use it here

File: client/src/index.html [hunk 5]
```diff
@@ -6,8 +6,10 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 </head>
-<body>
+<body class="mat-typography">
```
Comment (SharonEizner) [inline client/src/index.html:12]: why?

File: client/src/app/game-console/game-console.component.html [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<div id="console-box">
+
+    <cdk-virtual-scroll-viewport  style="height: 100%" itemSize="25" >
```
Comment (SharonEizner) [inline client/src/app/game-console/game-console.component.html:3]: nice 
and move the style to scss

File: client/src/app/board-game/board-game.component.ts [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+        this.horizontalWalls.push(wall);
+      }
+    }
+  }
+
+  private initActiveCells(): void {
+    for (let i = 0; i < 9; i++) {
+      this.activeCells.push([]);
+      for (let j = 0; j < 9; j++) {
+        this.activeCells[i].push(false);
```
Comment (RavivGuy) [inline client/src/app/board-game/board-game.component.ts:68]: where do you use j?

**Review by RavivGuy — APPROVED** [review]

**Pull request merged**